README.md

```diff
@@ -4,22 +4,44 @@ The k8s Terraform provider enables Terraform to deploy Kubernetes resources. Unl
 
 This project is a maintained fork of [ericchiang/terraform-provider-k8s](https://github.com/ericchiang/terraform-provider-k8s).
 
-## Usage
+## Installation
+
+### The Go Get way
 
 Use `go get` to install the provider:
 
 ```
 go get -u github.com/banzaicloud/terraform-provider-k8s
 ```
 
-Register the plugin in `~/.terraformrc`:
+Register the plugin in `~/.terraformrc` (see [Documentation]( https://www.terraform.io/docs/commands/cli-config.html ) for Windows users): 
 
 ```hcl
 providers {
   k8s = "/$GOPATH/bin/terraform-provider-k8s"
 }
 ```
 
+### The Terraform Plugin way  (enable versioning)
+
+Download a release from the [Release page]( https://github.com/banzaicloud/terraform-provider-k8s/releases ) and make sure the name matches the following convention
+
+| OS      | version | Name                              |
+| ------- | ------- | --------------------------------- |
+| LINUX   | 0.4.0   | terraform-provider-eks_v0.4.0     |
```

Comment:
I think it  would make sense if these names were real (using `k8s`)

Reply:
Also in link with my comment on issue #6